source/plugin/plugin-class.rst

```diff
@@ -58,8 +53,8 @@ game. In the example below, ``onServerStart()`` is called when the ``GameStarted
     import org.spongepowered.api.event.Listener;
     import org.spongepowered.api.event.game.state.GameStartedServerEvent;
 
-    @Plugin(id = "example", name = "Example Project", version = "1.0")
-    public class ExampleProject {
+    @Plugin(id = "exampleplugin", name = "Example Plugin", version = "1.0")
+    public final class ExamplePlugin {
```

Comment:
Was referring to this https://github.com/SpongePowered/SpongeDocs/pull/456#discussion_r59643422. If anything, they should both be consistent.